buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -183,10 +186,10 @@ class DistributedPerformanceTest extends PerformanceTest {
             requestContentType: ContentType.XML,
             body: buildRequest
         )
-        if (!response.success) {
```

Comment:
Why did you remove this? Does client.post already throw an exception when the build couldn't be scheduled?

Reply:
It does